static/app/views/performance/transactionSummary/transactionSpans/spanDetails/exclusiveTimeChart.tsx

```diff
@@ -152,8 +155,14 @@ export default function ExclusiveTimeChart(props: Props) {
                 right: 10,
                 top: 5,
                 selected: getSeriesSelection(location),
+                formatter: getExclusiveTimeDisplayedValue,
```

Comment:
I don't think it's necessary to add a custom formatter to the legend if you have already fixed the series name since echarts uses the series names to generate the legend.

Reply:
oh nice, fixed!